Yosemite/YosemiteTests/Stores/ProductShippingClassStoreTests.swift

```diff
@@ -134,6 +134,58 @@ final class ProductShippingClassStoreTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    func test_synchronizing_product_shipping_classes_of_the_same_page_size_has_next_page() {
+        // Arrange
+        let store = ProductShippingClassStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
+        network.simulateResponse(requestUrlSuffix: "products/shipping_classes", filename: "product-shipping-classes-load-all")
+
+        // Action
+        // The `pageSize` must match the response size in `product-shipping-classes-load-all.json`.
+        let pageSize = 3
+        var result: Result<Bool, Error>?
+        waitForExpectation { expectation in
+            let action = ProductShippingClassAction
+                .synchronizeProductShippingClassModels(siteID: sampleSiteID, pageNumber: defaultPageNumber, pageSize: pageSize) { aResult in
+                    result = aResult
+                    expectation.fulfill()
+            }
+            store.onAction(action)
+        }
+
+        // Assert
+        guard case let .success(hasNextPage) = result else {
+            XCTFail("Unexpected result: \(String(describing: result))")
+            return
+        }
+        XCTAssertTrue(hasNextPage)
```

Comment:
This may be enough:

```suggestion
        XCTAssertTrue(try XCTUnwrap(result).get())
```

It's not explicit but I think it's okay. ¯\\\_(ツ)\_/¯

Reply:
nice TIL'ed `Result.get()`! it shortens the assert code quite a bit, updated in https://github.com/woocommerce/woocommerce-ios/pull/2748/commits/397b12849fe849982b584d115c12ebba2a8a4734